examples/simple-app/package.json

```diff
@@ -1,13 +1,18 @@
 {
   "name": "simple-app",
   "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/avkvirtru/virtuoso-design-system.git"
+  },
+  "license": "ISC",
   "scripts": {
     "start": "parcel serve --no-cache index.html"
   },
   "dependencies": {
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
-    "virtuoso-design-system": "^1.0"
+    "virtuoso-design-system": "^1.3.1"
```

Comment:
@avkvirtru should this be `@latest` since we don't want to update all the time? Or `^1.3` so minors and patches are included.

Reply:
oh yeah! I like `latest`… will update